libtransact/src/contract/address/triple_key_hash.rs

```diff
@@ -43,9 +43,9 @@ impl Addresser<(String, String, String)> for TripleKeyHashAddresser {
         if (self.prefix.len() + self.first_hash_length + self.second_hash_length + last_hash_length)
             != ADDRESS_LENGTH
         {
-            return Err(AddresserError::TripleKeyHashAddresserError(
-                "Invalid hash length".to_string(),
-            ));
+            return Err(AddresserError {
```

Comment:
Similar to the double hash addresser, this doesn't really seem to make sense (it just verifies the math above it).

Reply:
Initially, this was the sole validation, makes sense to remove it now.